includes/blast_ui.admin.inc

```diff
@@ -156,6 +164,100 @@ function blast_ui_admin_form_validate($form, &$form_state) {
 
 }
 
+
+/*
+**
+** custom method disabled the web services 
+**
+*/
+function disable_web_services()
+{
+    $ar=array('services', 'rest_server');
+    $result = module_disable($ar,true);
+    try{
+    $eid = 1;
+    $num_deleted = db_delete('services_endpoint')->condition('eid', $eid)->execute();  
```

Comment:
Here we are just deleting the first service endpoint, correct? That seems risky since some Tripal sites already use Drupal services and may have their own custom endpoints before installing blast. It would be aweful if we deleted! Perhaps it would be best if we stored the eid of the service endpoint created in the enable function in a drupal variable. Then we could variable_get() it here to ensure we only delete our own service endpoint.